libminifi/src/sitetosite/SiteToSiteClient.cpp

```diff
@@ -499,6 +505,12 @@ int16_t SiteToSiteClient::send(std::string transactionID, DataPacket *packet, co
       return -1;
     }
     packet->_size += len;
+  } else if (flowFile && !flowfile_has_content) {
+    ret = transaction->getStream().write(len);  // Indicate zero length
+    if (ret != 8) {
+      logger_->log_debug("ret != 8");
```

Comment:
This log statement needs context. I don't think ret != 8 isn't sufficient for debugging

Reply:
Agreed. Just copy-pasted before, fixed all occurrences now to provide context.